easybuild/easyblocks/generic/intelbase.py

```diff
@@ -111,6 +112,42 @@ def __init__(self, *args, **kwargs):
 
         self.install_components = None
 
+    def _get_guesses_tools(self):
```

Comment:
@damianam I wouldn't make these hidden/private, since that is an indication these are/should only be used inside `IntelBase`, which is the opposite of what you're doing...

So, use `def get_guesses_tools(self):`

Reply:
Yep, I was a bit confused on the exact meaning of the underscore. I mean, I want these methods to be called exclusively by classes that inherited from `IntelBase`. But I think python don't have that kind of distinction.

Reply:
The leading underscore is only a convention too, it doesn't actually prevent you to do anything, it's just an indication that you shouldn't.

I don't think there's a way to restrict these methods to only be called by specific easyblocks, at least not without making an `IntelTools` that sits on top of `IntelBase`...